doc/api/http2.md

```diff
@@ -2,8 +2,6 @@
 
 <!--introduced_in=v8.4.0-->
```

Comment:
The docs state when a module was introduced, but do not state when it was introduced out of experimental.
Any ideas how to improve this? /cc @nodejs/documentation 
This is not blocking this PR from laanding and it shouldn't depend on it, that is more like a related issue.

Reply:
Should likely include it in the per-section changelog. I can update this to do so